### PR TITLE
Fix MacOS builds on Kokoro, after Kokoro update to Mojave

### PR DESCRIPTION
**Pull request by hevrard**
Kokoro experimentation, do not review.

### CODE REVIEW

File: kokoro/macos/package.sh [hunk 1]
```diff
@@ -70,7 +70,10 @@ iconutil -c icns -o GAPID.app/Contents/Resources/GAPID.icns GAPID.iconset
 pip install --upgrade --user dmgbuild pyobjc-framework-Quartz
 cp "$SRC"/background*.png .
 cp "$SRC/dmg-settings.py" .
-~/Library/Python/2.7/bin/dmgbuild -s dmg-settings.py GAPID gapid-$VERSION-macos.dmg
+# This path to dmgbuild (installed via pip --user) may change with Kokoro MacOS
+# updates. Value of 'site.USER_BASE' can give a hint of where to look, but
+# beware that default Python version may be different from the pip one.
+/Users/kbuilder/Library/Python/3.7/bin/dmgbuild -s dmg-settings.py GAPID gapid-$VERSION-macos.dmg
```
Comment (pmuetschard) [inline kokoro/macos/package.sh]: use `~/Library...` so that the script at least has a chance of working outside of the buildbots.

Comment (hevrard) [reply kokoro/macos/package.sh]: Done

**Comment by hevrard** [comment]
> Does not look like @pmuetschard s comment has actually been addressed

I added commit 4951c1f which makes that change, can you see it? Commit message is "Apply review comments"

**Comment by AWoloszyn** [comment]
Aha, it was not there when I was adding the comment.

**Pull request merged**